src/uct/tcp/tcp_iface.c

```diff
@@ -97,6 +97,7 @@ static ucs_status_t uct_tcp_iface_query(uct_iface_h tl_iface, uct_iface_attr_t *
                              UCT_IFACE_FLAG_EVENT_SEND_COMP  |
                              UCT_IFACE_FLAG_EVENT_RECV;
 
+    attr->listen_port      = ntohs(iface->config.ifaddr.sin_port);
```

Comment:
was tcp iface created in client/server mode?

Reply:
No. Since the tcp iface also has a listening port, it can be its iface attr.
Currently it won't return as part of ucp_listener_query but it will once tcp is used for conn establishment via sockaddr

Reply:
i think it's wrong to return a listening port if the iface was not created with server mode, because it's something internal, and not related to how the API is (or should be) used